src/app/shell/personal-cabinet/provider/create-workshop/create-workshop-address/create-workshop-address.component.ts

```
@@ -1,78 +1,231 @@
+      longitude: new FormControl('')
+    });
+  }
+
+  public createSearchFormGroup(): FormGroup {
+    return this.formBuilder.group({
+      settlementSearch: new FormControl('', FormValidators.defaultSearchValidators),
+      settlement: new FormControl('')
+    });
+  }
+
+  public deleteFormField(Array: FormArray, index: number): void {
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)